### DISCUSSION TITLE
Kamstrup omnipower Trollhättan Energi (Sweden ;-)

### DISCUSSION THREAD

**Discussion by rogerthn2019** [UtilitechAS/amsreader-firmware#882] [General]
Currently (2 minutes ago) data.json have "ic":882995.838 while the meter is showing 88301.06.
Yesterday during hour 16-17 the data.json value went from 88244.625 to 882448.635
Hmmm???
Right now data.json have 883017.405 and the meter 88301.99
Details below
![image](https://github.com/user-attachments/assets/ae1a534d-d417-40cc-aa38-3eb927ceabe4)

**Comment by rogerthn2019** [comment]
Now there seems to be a factor ~10 comparing with the, still attached S0 that has 2,302, 2,655 and 2,810 for those hours
![image](https://github.com/user-attachments/assets/608a964f-9516-4e55-8acf-4ea6be57b8f3)

**Reply by gskjold** [reply]
Unfortunately the payload from Kamstrup meters does not specify scaling, so firmware is not able to know what unit or how many decimals there is. Under meter configuration, tick the multiplier checkbox and set kwh multiplier to 0.1

**Reply by rogerthn2019** [reply]
Thanks, update done.
I'll report back

**Comment by ArnieO** [comment]
To explain this, we wrote the article "**_Kamstrup Omnipower in Sweden: 10x kWh error_**" some months ago: https://www.amsleser.no/module/ets_blog/blog?id_post=35

**Reply by rogerthn2019** [reply]
Looks like Trollhättan Energi applied the change between 16:00 and 17:00 on 2024-12-09

**Comment by rogerthn2019** [comment]
Is it possible to edit Tariff peaks?
![image](https://github.com/user-attachments/assets/c58e2bda-33f8-4c77-8b2a-ce28ec9dd005)

Maybe I haven't done enough searching?

**Reply by ArnieO** [reply]
Not directly in the GUI, but you can do it "manually" by downloading a backup file, edit it and load it back. 
Status/info screen > Backup & restore

**Reply by rogerthn2019** [reply]
Thanks, we'll see if I gone give it a try
